### PR TITLE
Mappings relating to code that uses CsvWriter

### PR DESCRIPTION
**Pull request by Sollace**
These are names I found whilst working on #1171 around where `ServerWorld` works with the CsvWriter.

- Changed the `exportX` methods to `dumpX` methods since that is the terminology used elsewhere, and fits with what Mojang seems to be doing.
- `BakedQuad` was missing mappings that I'm sure were mapped in 1.15, but weren't in this version

- Some names as per Mojang:

ServerChunkManager
    `method_21694` -> `getPendingTasks`
ServerTickScheduler
    `method_20825`  -> `getTicks`

```
writer.write(String.format("entities: %d\n", this.entitiesById.size()));
writer.write(String.format("block_entities: %d\n", this.field_9231.size()));
writer.write(String.format("block_ticks: %d\n", this.method_14196().getTicks()));
writer.write(String.format("fluid_ticks: %d\n", this.method_14179().getTicks()));
writer.write("distance_manager: " + lv.getTicketManager().getDebugString() + "\n");
writer.write(String.format("pending_tasks: %d\n", this.method_14178().getPendingTasks()));
```

`method_14196`, `method_14179`, `method_14178` don't show up as mappable in Enigma, so I couldn't confirm whether they have names or not.

That's about it, really.

### CODE REVIEW

File: mappings/net/minecraft/client/render/model/BakedQuad.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_777 net/minecraft/client/render/model/BakedQuad
+	FIELD field_22441 directionalLighting Z
```
Comment (liach) [inline mappings/net/minecraft/client/render/model/BakedQuad.mapping]: Is this like the `"gui_light": "side"` in item model? I feel this name itself isn't very clear and might get some javadocs.

File: mappings/net/minecraft/server/world/ChunkTicketManager.mapping [hunk 2]
```diff
@@ -69,6 +69,7 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 		ARG 4 argument
 	METHOD method_21623 getTicket (J)Ljava/lang/String;
 		ARG 1 pos
+	METHOD method_21683 getDebugString ()Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/server/world/ChunkTicketManager.mapping]: I'd suggest `getDebugMessage` or `getDumpMessage` if this is only used for dumping. `getString` doesn't sound very informative...

File: mappings/net/minecraft/test/StructureTestUtil.mapping [hunk 3]
```diff
@@ -38,3 +44,14 @@ CLASS net/minecraft/class_4525 net/minecraft/test/StructureTestUtil
...
+		ARG 0 altitude
+		ARG 1 pos
+		ARG 2 world
+	METHOD method_22369 createStructure (Ljava/lang/String;Lnet/minecraft/class_3218;)Lnet/minecraft/class_3499;
+		ARG 0 structureId
+		ARG 1 world
+	METHOD method_23646 createArea (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;I)Lnet/minecraft/class_3341;
+		ARG 0 pos1
+		ARG 1 pos2
+		ARG 2 horSize
```
Comment (liach) [inline mappings/net/minecraft/test/StructureTestUtil.mapping]:
```suggestion
		ARG 0 startPos
		ARG 1 endPos
		ARG 2 horizontalSize
```

File: mappings/net/minecraft/world/BlockRenderView.mapping [hunk 4]
```diff
@@ -6,6 +6,9 @@ CLASS net/minecraft/class_1920 net/minecraft/world/BlockRenderView
 	METHOD method_23752 getColor (Lnet/minecraft/class_2338;Lnet/minecraft/world/level/ColorResolver;)I
 		ARG 1 pos
 		ARG 2 colorResolver
+	METHOD method_24852 getBrightness (Lnet/minecraft/class_2350;Z)F
+		ARG 1 direction
+		ARG 2 hasDirectionalLighting
```
Comment (liach) [inline mappings/net/minecraft/world/BlockRenderView.mapping]:
```suggestion
		ARG 2 directionalLighting
```

**Review by liach — COMMENTED** [review]
Looks :+1: good!

File: mappings/net/minecraft/server/world/ChunkTicketManager.mapping (hunk 2 above)
Comment (Juuxel) [reply mappings/net/minecraft/server/world/ChunkTicketManager.mapping]: +1 for `getDebugMessage`

Comment (Sollace) [reply mappings/net/minecraft/server/world/ChunkTicketManager.mapping]: In this case the string it returns is describing the contents of the tick manager. So really, it's a bit like a `toString` method for dumps. `toDumpString`?

Comment (liach) [reply mappings/net/minecraft/server/world/ChunkTicketManager.mapping]:
```suggestion
	METHOD method_21683 toDumpString ()Ljava/lang/String;
```

File: mappings/net/minecraft/client/render/model/BakedQuad.mapping (hunk 1 above)
Comment (Sollace) [reply mappings/net/minecraft/client/render/model/BakedQuad.mapping]: That's a very good question. I tried to trace where it was coming from, and other parts refer to it as "shade/d" (which I will take responsibility for)

Neither name is idea, so i may look into it a little further to see if I can trace it back to an actual value in the json resources.

Comment (Sollace) [reply mappings/net/minecraft/client/render/model/BakedQuad.mapping]: The value traces back to `ModelElement$Deserializer` where the member name is called "shade" so I've changed everything to use that term.

```
            if (jsonObject5.has("shade") && !JsonHelper.hasBoolean(jsonObject5, "shade")) {
                throw new JsonParseException("Expected shade to be a Boolean");
            }
            boolean boolean10 = JsonHelper.getBoolean(jsonObject5, "shade", true);
```

Comment (Sollace) [reply mappings/net/minecraft/client/render/model/BakedQuad.mapping]: Perhaps it still needs a javadoc, because shade isn't exactly a clear meaning...

**Review by Juuxel — APPROVED** [review]

File: mappings/net/minecraft/server/world/ChunkTicketManager.mapping [hunk 5]
```diff
@@ -69,6 +69,7 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 		ARG 4 argument
 	METHOD method_21623 getTicket (J)Ljava/lang/String;
 		ARG 1 pos
+	METHOD method_21683 toDumpString ()Ljava/lang/String;
```
Comment (Runemoro) [inline mappings/net/minecraft/server/world/ChunkTicketManager.mapping:72]: What's a dump string? How is different from `getDebugString`?

Comment (liach) [reply mappings/net/minecraft/server/world/ChunkTicketManager.mapping:72]: a dumped string, as shown in debug command dump.

File: mappings/net/minecraft/test/StructureTestUtil.mapping [hunk 6]
```diff
@@ -38,3 +44,14 @@ CLASS net/minecraft/class_4525 net/minecraft/test/StructureTestUtil
...
+		ARG 0 altitude
+		ARG 1 pos
+		ARG 2 world
+	METHOD method_22369 createStructure (Ljava/lang/String;Lnet/minecraft/class_3218;)Lnet/minecraft/class_3499;
+		ARG 0 structureId
+		ARG 1 world
+	METHOD method_23646 createArea (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;I)Lnet/minecraft/class_3341;
+		ARG 0 startPos
+		ARG 1 endPos
+		ARG 2 horizontalSize
```
Comment (Runemoro) [inline mappings/net/minecraft/test/StructureTestUtil.mapping]: What does "area" mean here? What's the "horizontal size" and why isn't it determined by start and end positions?

Comment (Sollace) [reply mappings/net/minecraft/test/StructureTestUtil.mapping]: The area is a `BlockBox` used for positioning structures. I've revised the parameters a bit, but basically the values are:

`pos` = Pos1 of the box. This is the `pos` parameter passed down from other methods.
`size` = Width/Height/Depth of the box (passed down as the size of the structure)
`margin` = Extra margin to expend the area by along the X+Z axis.

The final box's dimensions are calculated as follows:
`min = pos - [margin, 3, margin]`
`max = pos + size + [margin - 1, 30, margin - 1]`

**Review by liach — APPROVED** [review]
Just change branch to 20w12a and this pr should be good to go.

**Comment by Sollace** [comment]
@liach Branch updated.

**Review by YanisBft — APPROVED** [review]

**Pull request merged**